certify-service/src/main/resources/application-local.properties

```diff
@@ -27,7 +27,7 @@ mosip.certify.authn.filter-urls={ '${server.servlet.path}/issuance/credential',
 mosip.certify.authn.issuer-uri=http://localhost:8088/v1/esignet
 mosip.certify.authn.jwk-set-uri=http://localhost:8088/v1/esignet/oauth/.well-known/jwks.json
 mosip.certify.authn.allowed-audiences={ '${mosipbox.public.url}${server.servlet.path}/issuance/credential', 'http://localhost:8088/v1/esignet/vci/credential' }
-
+mosip.certify.well-known.file.uri=https://raw.githubusercontent.com/vharsh/inji-certify/injicert-248-simplify/certify-service/src/main/resources/openid4vci-wk.json
```

Comment:
I'll update this to point to the inji-config repo once the PR gets merged there.